rosidl_generator_rs/resource/action.rs.em

```diff
@@ -90,15 +66,15 @@ pub struct @(type_name);
 
 impl rosidl_runtime_rs::Action for @(type_name) {
   // --- Associated types for client library users ---
-  type Goal = crate::@(subfolder)::@(type_name)@(ACTION_GOAL_SUFFIX);
-  type Result = crate::@(subfolder)::@(type_name)@(ACTION_RESULT_SUFFIX);
-  type Feedback = crate::@(subfolder)::@(type_name)@(ACTION_FEEDBACK_SUFFIX);
+  type Goal = @(type_name)@(ACTION_GOAL_SUFFIX);
+  type Result = @(type_name)@(ACTION_RESULT_SUFFIX);
+  type Feedback = @(type_name)@(ACTION_FEEDBACK_SUFFIX);
 
   // --- Associated types for client library implementation ---
-  type FeedbackMessage = crate::@(subfolder)::rmw::@(type_name)@(ACTION_FEEDBACK_MESSAGE_SUFFIX);
-  type SendGoalService = crate::@(subfolder)::rmw::@(type_name)@(ACTION_GOAL_SERVICE_SUFFIX);
+  type FeedbackMessage = super::@(subfolder)::@(type_name)@(ACTION_FEEDBACK_MESSAGE_SUFFIX);
+  type SendGoalService = super::@(subfolder)::@(type_name)@(ACTION_GOAL_SERVICE_SUFFIX);
   type CancelGoalService = action_msgs::srv::rmw::CancelGoal;
-  type GetResultService = crate::@(subfolder)::rmw::@(type_name)@(ACTION_RESULT_SERVICE_SUFFIX);
+  type GetResultService = super::@(subfolder)::@(type_name)@(ACTION_RESULT_SERVICE_SUFFIX);
```

Comment:
A lot of the changes look like this, simply replacing `crate::` with `super::`. However, when we referenced external to this module types, (i.e. `action_msgs::srv::rmw::CancelGoal`) we will need to depend on another mechanism such as a using statement or pulling it in as a separate crate to resolve the module path.